driver/src/main/java/org/neo4j/driver/v1/GraphDatabase.java

```diff
@@ -168,6 +169,8 @@ public static Driver driver( URI uri, AuthToken authToken, Config config )
         {
         case "bolt":
             return new DirectDriver( address, connectionSettings, securityPlan, poolSettings, config.logging() );
+        case "bolt+discovery":
```

Comment:
Personal preference to avoid `+` or other strange symbols in a url.
e.g.
`https://github.com/neo4j/neo4j-java-driver/pull/206/files` 
is far more better than 
`http+ssl://6ithub.com?!+neo4jneo4j+java\\driver+pull@@206~files`.